### PR TITLE
Update to Kube 1.18 / Go 1.13

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

File: pkg/cmd/infra/router/template.go [hunk 1]
```diff
@@ -401,7 +402,6 @@ func (o *TemplateRouterOptions) Run(stopCh <-chan struct{}) error {
 			Anonymous:               true,
 			TokenAccessReviewClient: tokenClient.TokenReviews(),
 			CacheTTL:                10 * time.Second,
-			ClientCAFile:            env("ROUTER_METRICS_AUTHENTICATOR_CA_FILE", ""),
```
Comment (ironcladlou) [inline pkg/cmd/infra/router/template.go:404]: This field disappeared and was replaced by something more complex, and we don't use it in the operator, so I simply deleted the usage.

File: pkg/router/controller/router_controller.go [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 import (
+	"context"
 	"fmt"
 	"sync"
 	"time"
 
 	kapi "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (Miciah) [inline pkg/router/controller/router_controller.go:10]: This change looks unnecessary (the package name is already "v1").

File: pkg/router/controller/factory/factory.go [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
 import (
+	"context"
 	"fmt"
 	"reflect"
 	"sort"
 	"time"
 
 	kapi "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (Miciah) [inline pkg/router/controller/factory/factory.go:11]: This change looks unnecessary.

File: pkg/router/controller/router_controller.go (hunk 2 above)
Comment (ironcladlou) [reply pkg/router/controller/router_controller.go:10]: gofmt adds it to disambiguate from `routev1 "github.com/openshift/api/route/v1"`.

File: pkg/router/controller/factory/factory.go (hunk 3 above)
Comment (ironcladlou) [reply pkg/router/controller/factory/factory.go:11]: gofmt compliance AFAICT

File: pkg/router/controller/router_controller.go (hunk 2 above)
Comment (ironcladlou) [reply pkg/router/controller/router_controller.go:10]: Actually, it could be gopls import organization. In any case, why not be explicit?

Comment (ironcladlou) [reply pkg/router/controller/router_controller.go:10]: (I will change it to better represent the package)

Comment (ironcladlou) [reply pkg/router/controller/router_controller.go]: Changed the import to `metav1`

File: pkg/router/controller/factory/factory.go (hunk 3 above)
Comment (ironcladlou) [reply pkg/router/controller/factory/factory.go]: Changed the import to `metav1`

**Comment by Miciah** [comment]
Did you run `go mod tidy`?  `go.sum` has a lot of different versions of the same packages...

**Comment by ironcladlou** [comment]
```
★ go version
go version go1.13.8 darwin/amd64

★ git status
On branch kube-1.18
Your branch is ahead of 'upstream/master' by 3 commits.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean

★ go mod tidy

★ git status
On branch kube-1.18
Your branch is ahead of 'upstream/master' by 3 commits.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
```

**Comment by Miciah** [comment]
I don't understand why `go.sum` has, for example, 20 entries for `golang.org/x/net`, but that's a separate matter.
/lgtm

**Comment by danehans** [comment]
Bootstrap failed to complete `level=error msg="Attempted to gather ClusterOperator status after installation failure: listing ClusterOperator objects: the server could not find the requested resource (get clusteroperators.config.openshift.io)"
`

/test e2e-aws

**Pull request merged**